cmd/kubeadm/app/cmd/token.go

```diff
@@ -229,7 +229,7 @@ func RunCreateToken(out io.Writer, client clientset.Interface, cfgPath string, c
 	// if --print-join-command was specified, print the full `kubeadm join` command
 	// otherwise, just print the token
 	if printJoinCommand {
-		joinCommand, err := cmdutil.GetJoinCommand(kubeConfigFile, internalcfg.BootstrapTokens[0].Token.String(), false)
+		joinCommand, err := cmdutil.GetJoinCommand(kubeConfigFile, internalcfg.BootstrapTokens[0].Token.String(), "", false, false, false)
```

Comment:
This signature is starting to not look so hot. Unfortunately I'm not sure I have suggestion beyond assigning these false values to variables and passing the variables in, but I'm lukewarm on that idea anyway. The only other suggestion is figure out a way to get rid of the bools from the signature, but that's a bigger task and probably outside the scope of this PR.